.github/workflows/fuzz-nightly.yml

```
@@ -0,0 +1,80 @@
+        run: |
+          ITER="${{ github.event.inputs.iterations || '1000' }}"
+          node scripts/validate-openapi.js --fuzz "$ITER" \
+            > reports-offline.txt 2>&1 || CODE=$?
+          echo "--- offline report ---"
+          tail -30 reports-offline.txt
+          echo "offline_exit=${CODE:-0}" >> "$GITHUB_OUTPUT"
+          exit "${CODE:-0}"
+
+      - name: Live conformance scan (if a base URL is available)
+        id: live
+        if: ${{ github.event.inputs.base_url != '' || secrets.BASE_URL != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)